detectable/src/main/java/com/blackduck/integration/detectable/detectables/yarn/YarnTransformer.java

```diff
@@ -50,10 +48,10 @@ public YarnTransformer(ExternalIdFactory externalIdFactory, EnumListFilter<YarnD
     public List<CodeLocation> generateCodeLocations(YarnLockResult yarnLockResult, List<NameVersion> externalDependencies)
         throws MissingExternalIdException {
         List<CodeLocation> codeLocations = new LinkedList<>();
-        logger.debug("Adding root dependencies for project: {}:{}", yarnLockResult.getRootPackageJson().getNameString(), yarnLockResult.getRootPackageJson().getVersionString());
+        logger.debug("Adding root dependencies for project: {}:{}, externalDeps: {}", yarnLockResult.getRootPackageJson().getNameString(), yarnLockResult.getRootPackageJson().getVersionString(), externalDependencies.size());
         LazyBuilderMissingExternalIdHandler lazyBuilderHandler = getLazyBuilderHandler(externalDependencies);
         ExternalIdDependencyGraphBuilder rootGraphBuilder = new ExternalIdDependencyGraphBuilder();
-        addRootDependenciesForProjectOrWorkspace(yarnLockResult, yarnLockResult.getRootPackageJson(), rootGraphBuilder);
+        addRootDependenciesForProject(yarnLockResult, yarnLockResult.getRootPackageJson(), rootGraphBuilder);
```

Comment:
Method addRootDependenciesForProject(...) belongs to 'ignore all workspaces' workflow.